docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1611,6 +1611,31 @@ How to **fix**:
 
   1. Once ingesters are stable, revert the temporarily config applied in the previous step.
 
+### MimirBlockBuilderNoCycleProcessing
+
+This alert fires when the block-builder stops reporting any processed cycles for an unexpectedly long time.
+
+How it **works**:
+
+- Block-builder periodically consumes a portion of the backlog from Kafka partition, and processes the consumed data into TSDB blocks. The block-buikder calls these periods "cycles".
+- If no cycles were processed during extended period of time, that can indicate that a block-builder instance is stuck and cannot complete cycle processing.
+
+How to **investigate**:
+
+- Check block-builder logs to see what its pods are busy with. Troubleshoot based on that.
+
+### MimirBlockBuilderCompactAndUploadFailed
+
+How it **works**:
+
+- Block-builder periodically consumes data from Kafka topic, and processes the consumed data into TSDB blocks.
```

Comment:
```suggestion
- The block-builder periodically consumes data from Kafka topic and processes the consumed data into TSDB blocks.
```

Reply:
Could we say either "a Kafka topic" or "the Kafka topic" depending on which is more accurate?